vs

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
+# This is a small wrapper that lets VegaStrike - Upon the Coldest Sea
+# run from seemingly anywhere
 set -e
 
 VEGASTRIKE_LOCAL_SHARE_DIR="/usr/local/share/vegastrike"
 VEGASTRIKE_SHARE_DIR="/usr/share/vegastrike"
 
-# This is a small wrapper that lets VegaStrike - Upon the Coldest Sea
-# run from seemingly anywhere
+#Let's keep a log of the console
+touch ~/.vegastrike/console.log
+mv -vf ~/.vegastrike/console.log ~/.vegastrike/console.bak 
+
 if [ -d "${VEGASTRIKE_LOCAL_SHARE_DIR}" ]; then 
-    vegastrike -d"${VEGASTRIKE_LOCAL_SHARE_DIR}"
+    vegastrike -d"${VEGASTRIKE_LOCAL_SHARE_DIR}" "$1" | tee ~/.vegastrike/console.log
```

Comment:
wouldn't `tee` cause it to be on the screen too?
either way, can we do this both for `stdout` and `stderr`?

Reply:
Yes, it leaves the console output to the screen, if there is a screen attached. We can modify it for stdout and err, if you want. Right now it captures _everything_ in it's glorious completeness.